docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
Basically it looks like we don't need it for our initial usecase (heapster). This may be more useful for the DerivedMetrics, and I'm not against adding it back. Do you anticipate it being used with RawMetrics?